src/static_templates/src/t2cl.rs

```diff
@@ -0,0 +1,135 @@
+                        register: CpuidRegister::Edx,
+                        bitmap: RegisterValueFilter {
+                            filter: 0b00000000000000000000000100011100,
+                            value: 0b00000000000000000000000000000000,
+                        },
+                    },
+                ],
+            },
+            CpuidLeafModifier {
+                leaf: 0xd,
+                subleaf: 0x0,
+                flags: KvmCpuidFlags(1),
+                modifiers: vec![CpuidRegisterModifier {
+                    register: CpuidRegister::Eax,
+                    bitmap: RegisterValueFilter {
+                        filter: 0b00000000000000000000001011111000,
+                        value: 0b00000000000000000000000000000000,
+                    },
+                }],
+            },
+            CpuidLeafModifier {
+                leaf: 0xd,
+                subleaf: 0x1,
+                flags: KvmCpuidFlags(1),
+                modifiers: vec![CpuidRegisterModifier {
+                    register: CpuidRegister::Eax,
+                    bitmap: RegisterValueFilter {
+                        filter: 0b00000000000000000000000000001110,
+                        value: 0b00000000000000000000000000000000,
+                    },
+                }],
+            },
+            CpuidLeafModifier {
+                leaf: 0x80000001,
+                subleaf: 0x0,
+                flags: KvmCpuidFlags(0),
+                modifiers: vec![
+                    CpuidRegisterModifier {
+                        register: CpuidRegister::Ecx,
+                        bitmap: RegisterValueFilter {
+                            filter: 0b00100000000000000000000111000000,
+                            value: 0b00000000000000000000000000000000,
+                        },
+                    },
+                    CpuidRegisterModifier {
+                        register: CpuidRegister::Edx,
+                        bitmap: RegisterValueFilter {
+                            filter: 0b00000111110000000000000000000000,
+                            value: 0b00000000000000000000000000000000,
+                        },
+                    },
+                ],
+            },
+            CpuidLeafModifier {
+                leaf: 0x80000008,
+                subleaf: 0x0,
+                flags: KvmCpuidFlags(0),
+                modifiers: vec![CpuidRegisterModifier {
+                    register: CpuidRegister::Ebx,
+                    bitmap: RegisterValueFilter {
+                        filter: 0b00000000000000000000001000000000,
+                        value: 0b00000000000000000000000000000000,
+                    },
+                }],
+            },
+        ],
+        msr_modifiers: vec![RegisterModifier {
+            addr: 0x10a,
+            bitmap: RegisterValueFilter {
+                filter: 0b0000000000000000000000000000000000000000000000000000000011101011,
```

Comment:
I think the filter should be all 1's, because we only set some bits and leave the rest as zeroes.

Reply:
You're right! Will fix.

Reply:
fixed